src/metaschema/oscal_implementation-common_metaschema.xml

```diff
@@ -579,7 +579,7 @@
                         &allowed-values-component_inventory-item_property-name;
 
                   </allowed-values>
-                  <allowed-values target="prop[has-oscal-namespace('http://csrc.nist.gov/ns/oscal') and @name='asset-type']/@value">
+                  <allowed-values allow-other="yes" target="prop[@name='asset-type']/value()">
```

Comment:
Blocking: I apologize I let this slip through when pairing over BlueJeans with the has-oscal-namespace() drop and maybe the non-blocking style recommendation on placing the allow-other in the order we usually do.

```suggestion
                  <allowed-values target="prop[has-oscal-namespace('http://csrc.nist.gov/ns/oscal') and @name='asset-type']/@value" allow-other="yes">
```

Reply:
Did you commit this third suggestion, @Arminta-Jenkins-NIST? We need the prop check with `has-oscal-namespace` here as well.